docs/book/src/clusterctl/commands/move.md

```diff
@@ -60,4 +60,6 @@ This can now be achieved with the following procedure:
 5. Get the kubeconfig for the new target management cluster
 6. Use `clusterctl init` with the new cluster's kubeconfig to install the provider components 
 7. Use `clusterctl move` to move the Cluster API resources from the bootstrap cluster to the target management cluster
-8. Delete the bootstrap cluster
\ No newline at end of file
+8. Delete the bootstrap cluster
+
+Please note that since the control plane nodes will have a `NoSchedule taint`, `clusterctl init` will not be able to schedule any pods and will timeout if the target cluster does not have any worker nodes.
```

Comment:
```suggestion
> Note: It's required to have at least one worker node to schedule Cluster API workloads (i.e. controllers).
> A cluster with a single control plane node won't be sufficient due to the `NoSchedule` taint. If a worker node isn't available, `clusterctl init` will timeout.
```

Reply:
Tried to flip the message to give actionable content, let me know what you think!

Reply:
yeah, sounds better that way!